serverless.ts

```diff
@@ -0,0 +1,50 @@
+const serverlessConfiguration = {
+    service: "product-service",
+    provider: {
+      name: 'aws',
+      runtime: 'nodejs20.x',
+      stage: 'dev',
+      region: 'eu-west-1',
+    },
+    plugins: [
+      'serverless-offline'
+  ],
+    functions: {
+        getProductsList: {
+          handler: '.webpack/getProductsList.getProductsList',
+          events: [
+            {
+              http: {
+                path: '/products',
+                method: 'get',
+                cors: {
```

Comment:
Why do you have cors setup for one method and not for second?

Reply:
Done